Sources/Client/Client/Client+Events.swift

```diff
@@ -45,29 +47,24 @@ public final class SubscriptionBag: Cancellable {
 }
 
 extension Client {
-    /// Observe events for the given event types.
+    
+    /// Observe events for the given client event types.
     /// - Parameters:
-    ///   - eventTypes: A set of event types to be observed. Defaults to all events.
+    ///   - eventTypes: A set of client event types to be observed. Defaults to all client events.
     ///   - callback: Callback closure to be called for each new event.
-    /// - Returns: `Subscription` object to be able to cancel observing. Call `subscription.cancel()` when you want to stop observing.
-    /// - Warning: Subscriptions do not cancel on `deinit` and that can cause crashes / memory leaks, so make sure you handle subscriptions correctly.
-    public func subscribe(forEvents eventTypes: Set<EventType> = Set(EventType.allCases),
-                          _ callback: @escaping OnEvent) -> Cancellable {
-        subscribe(forEvents: eventTypes, cid: nil, callback)
+    /// - Returns: `Subscription` object to be able to cancel observing.
+    ///            Call `subscription.cancel()` when you want to stop observing.
+    /// - Warning: Subscriptions do not cancel on `deinit` and that can cause crashes / memory leaks,
+    ///            so make sure you handle subscriptions correctly.
+    public func subscribe(forClientEvents eventTypes: Set<ClientEventType> = Set(ClientEventType.allCases),
```

Comment:
I don't think `forClientEvents` is needed as argument label, it's clear that we're subscribing to Client events from the call:
```swift
Client.shared.subscribe(forEvents:)
```